### DISCUSSION TITLE
Cant take a photo of school portal via iphone back camera.

### DISCUSSION THREAD

**Discussion by whiskercodes** [community/community#144919] [GitHub Education]
### Hi everyone,

Cant take a photo of school portal via iphone back camera, i was trying to verify for the github education prorgram, but only the front camera works there for its SUPER hard to take an image of the school portal

**Comment by FeiSuoerDe** [comment]
You can try shooting with a computer

**Reply by FeiSuoerDe** [reply]
Forgive me, I mean you can try disabling the front camera, I don't know if your device can do that, or try a different browser, as the community says, and usually submit a message a lot of times. I tried a full 19 times

**Reply by whiskercodes** [reply]
The problem is that file uploads dosent exist, there for this is super hard to do.

**Reply by FeiSuoerDe** [reply]
Well, for now, the official has turned off file uploading, so you can only print your documents and take a photo with your hand

**Comment by toster234** [comment]
Same problem 👍

**Comment by drfrostongithub** [comment]
Ye, I had this problem too, cannot even report it anywhere. This is suck.
